app/views/shared/_footer.html.erb

```diff
@@ -21,6 +21,11 @@
               Privacy
             </a>
           </li>
+          <li>
+            <a href="https://do.co/hacktoberbrand">
```

Comment:
I'm assuming this is meant to go to a figma currently?

Reply:
Yeah, same as what's on the live site